api/odigos/v1alpha1/instrumentationconfig_types.go

```diff
@@ -101,10 +96,63 @@ type HeadSamplingConfig struct {
 	FallbackFraction float64 `json:"fallbackFraction"`
 }
 
+// The InstrumentationLibraryCapabilityParameter represents a single configuration property that can be set for an instrumentation library option.
+// For example, the capability might be payload collection, and the parameter can be the maximum size of the payload to record,
+// if to skip recording incomplete payloads, conditions for recording payload (for example only for specific mime types - "only application/json"),
+// additional attributes to record original length, etc.
+type InstrumentationLibraryCapabilityParameter struct {
+	// The name of the property that should be configured for the instrumentation library option
+	ParameterName string `json:"parameterName"`
```

Comment:
What is the difference between this and `CapabilityName`?
If possible, adding an example in the comments will make it clearer